src/renderer/dx/CustomTextLayout.cpp

```diff
@@ -129,13 +129,30 @@ CustomTextLayout::CustomTextLayout(gsl::not_null<IDWriteFactory1*> const factory
         // Allocate enough room to have one breakpoint per code unit.
         _breakpoints.resize(_text.size());
 
-        // Call each of the analyzers in sequence, recording their results.
-        RETURN_IF_FAILED(_analyzer->AnalyzeLineBreakpoints(this, 0, textLength, this));
-        RETURN_IF_FAILED(_analyzer->AnalyzeBidi(this, 0, textLength, this));
-        RETURN_IF_FAILED(_analyzer->AnalyzeScript(this, 0, textLength, this));
-        RETURN_IF_FAILED(_analyzer->AnalyzeNumberSubstitution(this, 0, textLength, this));
-        // Perform our custom font fallback analyzer that mimics the pattern of the real analyzers.
-        RETURN_IF_FAILED(_AnalyzeFontFallback(this, 0, textLength));
+        const bool whiteSpacesOnly = std::all_of(
+            _text.begin(), _text.end(), [](wchar_t c) noexcept {

```

Comment:
Out of curiosity, can we just pass `&iswspace` as the function? (my c++17-fu is not strong so this is mostly an educational question).

(and the same below)